source/_patterns/02-molecules/blocks/alert/alert.js

```diff
@@ -1,8 +1,10 @@
-document.getElementById("alert-close").onclick = function alertClose() {
-  var alert = document.getElementById("alert-bar");
-  if (alert.style.display === "none") {
-    alert.style.display = "block";
-  } else {
-    alert.style.display = "none";
-  }
-};
+function alertCloseclick() {
+  document.getElementById("alert-close").onclick = function alertClose() {
```

Comment:
if there are multiple alerts in a page, Id would only work on the first one.

Reply:
Trial court solely uses the close link and there should only be one per page